sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -60,7 +60,7 @@ AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
             {
               ._internal = {
                 .process = az_http_pipeline_policy_apiversion,
-                .p_options= &self->_internal.options._internal.api_version,
+                .p_options= &options->_internal.api_version,
```

Comment:
Please put these back the way they were. Note that the clients make a copy of the options so that the customer doe not have to keep them around for us. This also ensures thread safety; the customer CAN modify the options struct before creating another client because each client makes its own copy of the options.